buddyboss/buddyboss-platform
# Unable to Translate Strings within JavaScript Files (2)

### All Submissions:






### Changes proposed in this Pull Request:
Steps to reproduce the behavior:


Fixes #456.

### How to test the changes in this Pull Request:
1. Translate every word through Loco Translate
2. Go to feed
3. Add one photo
4. See text "Drop files here to upload" translated

### Proof Screenshots or Video
http://prntscr.com/qnfdkn



### Other information:
